.github/workflows/ci.yml

```diff
@@ -54,3 +52,24 @@ jobs:
 
       - name: Run tests
         run: npm test --ignore-scripts
+
+  smoketest:
+    runs-on: ubuntu-latest
+    # if: ${{ github.actor != 'dependabot[bot]' }}
+    strategy:
+      matrix:
+        regex: []
```

Comment:
Why'd you comment this out entirely? I was going to say "I don't think this matrix is right" (I think you want to include only items 1 and 3, not 2 and 4) but curious why you're removing it entirely?

Reply:
oops never mind, this is outdated :)